packages/feedback/README.md

```diff
@@ -41,59 +46,159 @@ Sentry.init({
+### Additional UI Customization
+Similar to theme customization above, these are additional CSS variables that can be overridden. Note these are not supported in the constructor.
 
-As an alternative to the NPM package, you can use Feedback as a CDN bundle.
-Please refer to the [Feedback installation guide](https://docs.sentry.io/platforms/javascript/session-feedback/#install) for CDN bundle instructions.
+| Variable | Default | Description |
+| --- | --- | --- |
+| `--bottom` | `1rem` | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--right` | `1rem` | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--top` | `auto` | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--left` | `auto` | By default the widget has a position of fixed, and is in the bottom right corner. |
+| `--z-index` | `100000` | The z-index of the widget |
+| `--font-family` | `"'Helvetica Neue', Arial, sans-serif"` | Default font-family to use|
+| `--font-size` | `14px` | Font size |
 
+### Event Callbacks
+Sometimes it’s important to know when someone has started to interact with the feedback form, so you can add custom logging, or start/stop background timers on the page until the user is done.
 
-## Configuration
+Pass these callbacks when you initialize the Feedback integration:
 
-### General Integration Configuration
+```javascript
+new Feedback({
+  onActorClick: () => {},
+  onDialogOpen: () => {},
+  onDialogClose: () => {},
+  onSubmitSuccess: () => {},
+  onSubmitError: () => {},
+});
+```
 
-The following options can be configured as options to the integration, in `new Feedback({})`:
+## Further Customization
+There are two more methods in the integration that can help customization.
 
-| key       | type    | default | description |
-| --------- | ------- | ------- | ----------- |
-| tbd       | boolean | `true`  | tbd         |
+### Bring Your Own Button
 
+You can skip the default widget button and use your own button. Call `feedback.attachTo()` to have the SDK attach a click listener to your own button. You can additionally supply the same customization options that the constructor accepts (e.g. for text labels and colors).
 
+```javascript
+const feedback = new Feedback({
+  // Disable injecting the default widget
+  autoInject: false,
+});
+
+feedback.attachTo(document.querySelector('#your-button'), {
+    formTitle: "Report a Bug!"
+});
+```
 
-## Manually Sending Feedback Data
+### Bring Your Own Widget
 
-Connect your own feedback UI to Sentry's You can use `feedback.flush()` to immediately send all currently captured feedback data.
-When Feedback is currently in buffering mode, this will send up to the last 60 seconds of feedback data,
-and also continue sending afterwards, similar to when an error happens & is recorded.
+You can also bring your own widget and UI and simply pass a feedback object to the `sendFeedback()` function.
```

Comment:
Might be nice for us to think about a way to make the widget work without js in the future via direct form submit, but def something for after GA (we can prob just open a feature request issue and see how many thumbs ups we get).